JS/Lesson31/savas/my-app/src/components/NewItem/index.jsx

```diff
@@ -0,0 +1,32 @@
+import {useState} from 'react';
+
+export const NewItem = ({ handleNewTodo }) => {
+    const [item, setItem] = useState("");
+    const handleInputChange = (event) => {
+      setItem(event.target.value);
+    };
+  
+    const handleSubmit = (event) => {
+      event.preventDefault();
+      if (item) {
+        handleNewTodo(item);
+        setItem("");
+      }
+    };
+  
+    return (
+      <>
```

Comment:
No need for fragment component (<>) here as you only have a <form> component on the most outer layer in the return statement

Reply:
Oh, I missed it :)